### PR TITLE
Improve vector store onboarding exp

### PR DESCRIPTION
**Pull request by jeffchuber**
This PR
- fixes the `similarity_search_by_vector` example, makes the code run and adds the example to mirror `similarity_search`
- reverts back to chroma from faiss to remove sharp edges / create a happy path for new developers. (1) real metadata filtering, (2) expected functionality like `update`, `delete`, etc to serve beyond the most trivial use cases 

@hwchase17

### CODE REVIEW

File: docs/docs_skeleton/static/img/vector_stores.jpg
Comment (baskaryan) [inline docs/docs_skeleton/static/img/vector_stores.jpg:1]: think separating 2 and 3 is a bit confusing, not obvious you query in order to retriever. maybe we could make it look more like a loop (query goes in, most similar comes out)

Comment (jeffchuber) [reply docs/docs_skeleton/static/img/vector_stores.jpg:1]: that's great feedback! 

what do you think of this?

Frame 2 (1)

Comment (baskaryan) [reply docs/docs_skeleton/static/img/vector_stores.jpg:1]: love it!

Comment (jeffchuber) [reply docs/docs_skeleton/static/img/vector_stores.jpg:1]: @baskaryan great! image pushed to branch

**Comment by jeffchuber** [comment]
@baskaryan anything else I can do to help here?

**Pull request merged**